conanfile.py

```diff
@@ -16,19 +17,22 @@ class FmtConan(ConanFile):
     exports_sources = ['CMakeLists.txt']
     generators = 'cmake'
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "header_only": [True, False]}
-    default_options = "shared=False", "header_only=False"
+    options = {"shared": [True, False], "header_only": [True, False], "fPIC": [True, False]}
+    default_options = "shared=False", "header_only=False", "fPIC=False"
```

Comment:
Actually we use `fPIC=True` by default, even for static library, you can check in our template: https://github.com/bincrafters/conan-templates/blob/master/conanfile.py#L28